recipes/stopwordsiso/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "stopwordsiso" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5be2b4f25afc296fca4f626710b98168949057f81a655e2dbdf670ddfce2af50
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+    - setuptools
```

Comment:
Are you sure `setuptools` is required?

Reply:
Yes, since it uses `import pkg_resources` to find the packaged "stopwords-iso.json". I checked, and the build failed without this dependency.